tools/python/transit/gtfs/download_gtfs.py

```diff
@@ -17,17 +18,39 @@
 MAX_RETRIES = 2
 MAX_SLEEP_TIMEOUT_S = 30
 
+RAW_FILE_MOBILITYDB = "raw_mobilitydb.csv"
+
 URLS_FILE_TRANSITLAND = "feed_urls_transitland.txt"
-URLS_FILE_OMD = "feed_urls_openmobilitydata.txt"
+URLS_FILE_MOBILITYDB = "feed_urls_mobilitydb.txt"
+
+URL_MOBILITYDB_GTFS_SOURCE = "https://bit.ly/catalogs-csv"
```

Comment:
Why do you use short URL instead of a full URL `https://storage.googleapis.com/storage/v1/b/mdb-csv/o/sources.csv?alt=media` ?

Reply:
The link to the CSV file on their website used a link shortener. I will update it with its target url.

Reply:
It creates an additional HTTP round-trip to resolve the shortener. I would document in the comment the original web page where you get that shortener, and use the resolved URL. If the scheme changes, we'll need to update the parser too.